src/stores/defaults.ts

```diff
@@ -52,7 +52,13 @@ export const storeStateDefault: StoreState = {
   ],
   currentMatchIndex: 1,
   matchIdToMatchState: {},
-  phoneNumberToContact: {},
+  phoneNumberToContact: {
+    phoneNumber: {
+      phoneNumber: '0987654321',
```

Comment:
Change:

    phoneNumber: {
       phoneNumber: '0987654321',

to:

    '+1987654321': {
       phoneNumber: '+1987654321',